### PR TITLE
Add Python samples for features described in the README

### LINKED ISSUE #4: Add Python code samples for each feature described in the README
The README for this repository shows how to use some of the new Redis features from version 6.2 using `redis-cli`.  Add a folder called `python` to this repo containing a single Python script that demonstrates each of these features using the same commands shown in the README.  Use the [redis-py](https://github.com/redis/redis-py) client for this, don't add any other dependencies.  Be sure to add a README in the `python` folder the describes how to run your code in a Python virtual environment, and to commit a `requirements.txt` file for use with `pip`.

This is part of our participation in Hacktoberfest 2022. Learn how to complete this issue here: https://redis.io/community/hacktoberfest/

Need help or want to talk to us? Join us on Discord where we have a dedicated Hacktoberfest channel: https://discord.gg/ueQwKUh5Z3

If you're interested in taking this issue on, please mention @simonprickett and @SuzeShardlow in the comments and ask to be assigned.

### PR DESCRIPTION
**Pull request by kavyabhat02**
Closes #4.

### CODE REVIEW

File: python/samples.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+r = redis.Redis(host='localhost', port=6379, db=0)
+
+##Strings: New Options for the SET command
+#Demo of the GET option
+print(r.set('favoriteflavor', "Mint Choc Chip"))
+print(r.set('favoriteflavor', "Vanilla Bean", get=True))
+print(r.get('favoriteflavor'))
+
+#Demo of the EXAT option
```
Comment (simonprickett) [inline python/samples.py:12]: I think it would be good for each demo to `print` out which demo it is, so that the output is easier to read.

**Review by simonprickett — COMMENTED** [review]
@kavyabhat02 left some minor comments / suggestions - this is very close to being ready, nice work thanks.

**Review by simonprickett — APPROVED** [review]
Thanks @kavyabhat02 looks good - @SuzeShardlow approved to merge.

**Comment by SuzeShardlow** [comment]
@kavyabhat02 thanks for collaborating with us!  If you're a member of our Discord server (https://discord.gg/Redis), please let us know your handle so we can give you the `Hacktoberfest 2022` role there :)

**Comment by kavyabhat02** [comment]
> @kavyabhat02 thanks for collaborating with us! If you're a member of our Discord server (https://discord.gg/Redis), please let us know your handle so we can give you the `Hacktoberfest 2022` role there :)

Thanks! My Discord handle is kavbhat#0810.

**Comment by SuzeShardlow** [comment]
Thanks @kavyabhat02, I've given you the `Hacktoberfest 2022` role in our Discord server :)

**Pull request merged**